internal/kittehs/strings.go

```diff
@@ -114,3 +114,28 @@ func (w *IndentedStringWriter) Write(p []byte) (n int, err error) {
 
 	return
 }
+
+func StringWithoutComments(s string) string {
+	var (
```

Comment:
```go
var commentRe = regexp.MustCompile(`(?m)#.*$`)
text = commentRe.ReplaceAllString(text, "")
```

Reply:
couldn't make it work - got some issue with newlines, also - i think what I've done is actually faster.